packages/react-client/README.md

```diff
@@ -12,19 +12,89 @@ npm install --save @telnyx/react-client
 
 ## Usage
 
-```tsx
-import React, { Component } from 'react';
+```jsx
+// App.jsx
+import { TelnyxClientProvider } from '@telnyx/react-client';
```

Comment:
We still call the main package's class `TelnyxRTC`, so I think we should got with `TelnyxRTCProvider`, `useTelnyxRTC`, and so on.